src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --color-hover: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.img {
+  display: flex;
+  height: 40px;
+  width: 40px;
```

Comment:
The `.img` class seems to be used for an image, but you've set it to `display: flex;`. This is usually unnecessary for an image. If you want to center the image, you could use `margin: auto;` or align the parent element. Also, setting both width and height can distort the image. It's better to set only one dimension and let the other adjust automatically to maintain the aspect ratio.